### PR TITLE
Remove more stuff from the work directory.

### PR DESCRIPTION
**Pull request by mvollmer**

### CODE REVIEW

File: hubbot [hunk 1]
```diff
@@ -415,6 +431,9 @@ def expire_work_dirs(github_visible_commits):
             if not dry_run:
```
Comment (stefwalter) [inline hubbot:431]: I know this is old code, but i would explicitly put a check that the directory being deleted is a subdirectory of the work directory or a relative directory, and not an absolute directory.

**Comment by stefwalter** [comment]
Other than that, looks good.

File: hubbot (hunk 1 above)
Comment (mvollmer) [inline hubbot:431]: It's always one of the elements returned by list_work_dirs(). Should we double check that list_work_dirs() works correctly?

**Comment by mvollmer** [comment]
Rebased.

**Pull request merged**